src/helpers/programHelper.js

```diff
@@ -920,6 +923,140 @@ class ProgramServiceHelper {
      */
     return _(programs).chain().sortBy((prg) => prg.createdon).sortBy((prg) => prg.matchCount).values().reverse();
   }
+
+  async getProgramsForContribution(data, filters){
+    const nomination = data.request.filters.nomination;
+    const user_id = _.get(nomination, 'user_id');
+    const organisation_id = _.get(nomination, 'organisation_id');
+    const status = _.get(nomination, 'status');
+    const roles = _.get(nomination, 'roles');
+
+    // Contributor org user (my projects)
+    if(user_id && organisation_id && status && roles) {
+      const res = await this.getContribUserPrograms(data, filters);
+      let aggregatedRes = [];
```

Comment:
Do this in a`try catch`

Reply:
Added try-catch in the parent function. If error any error then the parent function will catch it and throw an error in API resp.